src/components/FilePreview/FilePreviewAction.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import type {FC, MouseEventHandler} from 'react';
+
+import {Button, Icon, IconData, Text, Tooltip} from '@gravity-ui/uikit';
+
+export interface FilePreviewActionProps {
+    id?: string;
+    icon: IconData;
+    title: string;
+    label?: string;
+    href?: string;
+    onClick?: MouseEventHandler<HTMLButtonElement | HTMLAnchorElement>;
+}
+
+export const FilePreviewAction: FC<FilePreviewActionProps> = ({
+    id,
```

Comment:
Generate this id in component, there is no point ti pass id that required only for a11y